php_1/db/PdoConnectionClass.php

```diff
@@ -1,16 +1,25 @@
 <?php
+require_once 'Config.php';
+
 class PdoConnectionClass
 {
-    private $server = "mysql:host=localhost;dbname=mydatabase3";
-    private $user = "root";
-    private $pass = "";
+    private $serverName = Config::HOST;
+    private $database = Config::DATABASE;
+    private $user = Config::USER;
+    private $pass = Config::PASSWORD;
```

Comment:
We can use directly the config constant variables in the Pdo Connection without the need of using private variables.